generators/app/index.js

```diff
@@ -35,57 +37,80 @@ module.exports = Generator.extend({
     return this.prompt(prompts).then(function (props) {
       options = _.assign(options, props);
     });
-
-    // disabling this for now as pattern lab starter v8 doesn't really have drupal 7 or drupal 8 differences
-    // this.composeWith('pattern-lab-starter:extras', {options: options}, {
-    //   local: path.resolve(__dirname, '../extras')
-    // });
-
   },
 
   configuring: function () {
 
   },
 
   default: function () {
-    var dest = options.themePath ? path.resolve(process.cwd(), options.themePath) : './';
+    const release = options['release'] || 'master'
+    const release_path = release + '.tar.gz';
+    const compressed = _.last(_.split(release_path, '/'));
+    const decompressed = _.replace('particle-' + release, '/', '-');
```

Comment:
Hit me with what's going on with `compressed` vs `decompressed` here. Splitting paths on a divider feels like something that should be going through a node fs function.

Reply:
If you name a branch something crazy, like epic/v10, then the resulting file download will be named `v10.tar.gz` (though it occurs to me we could use curl's -o flag and standardize that).  Compressed is the name of that downloaded file, while decompressed is the name of the directory after you decompress the tarball, and that file has had it's slashes hyphenized.